### PR TITLE
fix: from nav to modals

### PR DESCRIPTION
**Pull request by arnaudambro**
- fix: migration
- ongoing

### CODE REVIEW

File: app/src/components/DrinkPersonalisation.js [hunk 1]
```diff
@@ -11,21 +11,25 @@ import ButtonPrimary from './ButtonPrimary';
...
+const DrinkPersonalisation = ({ updateDrinkKey, hide, quantitySelected, setQuantitySelected }) => {
   const route = useRoute();
   const timestamp = route?.params?.timestamp;
 
   const onSetQuantity = (q) => {
     setQuantity(q);
   };
   const [ownDrinksCatalog, setOwnDrinksCatalog] = useRecoilState(ownDrinksCatalogState);
-  const [showModal, setShowModal] = useState(false);
+  const [showModalUdate, setShowModal] = useState(false);
```
Comment (axelgcd) [inline app/src/components/DrinkPersonalisation.js:24]: showModalUdate => showModalUpdate

File: app/src/components/DrinkPersonalisation.js [hunk 2]
```diff
@@ -232,23 +236,28 @@ const DrinkPersonalisation = ({ navigation, quantitySelected, setQuantitySelecte
         )}
       </View>
       <ModalUpdateSuppressionDrink
-        isUpdate={isUpdateWanted}
-        showModal={showModal}
+        updateDrinkKey={isUpdateWanted}
+        visible={showModalUdate}
```
Comment (axelgcd) [inline app/src/components/DrinkPersonalisation.js:240]: showModalUdate => showModalUpdate

File: app/src/scenes/ConsoFollowUp/ConsoFeedDisplay.js [hunk 3]
```diff
@@ -30,6 +30,9 @@ const ConsoFeedDisplay = ({
 
   const drink = consolidatedCatalog.find((catalogDrink) => catalogDrink.drinkKey === drinkKey);
   const Icon = getIcon(drink.icon);
+  console.log('drink', drink);
```
Comment (axelgcd) [inline app/src/scenes/ConsoFollowUp/ConsoFeedDisplay.js:33]: clean logs

File: app/src/scenes/ConsoFollowUp/ConsoFeedDisplay.js [hunk 4]
```diff
@@ -30,6 +30,9 @@ const ConsoFeedDisplay = ({
...
+  console.log('Icon', Icon);
+  console.log('drinkName', drinkName);
   return (
```
Comment (axelgcd) [inline app/src/scenes/ConsoFollowUp/ConsoFeedDisplay.js:36]: clean logs

**Review by axelgcd — COMMENTED** [review]
Review comments

File: app/src/components/DrinkPersonalisation.js (hunk 2 above)
Comment (arnaudambro) [reply app/src/components/DrinkPersonalisation.js:240]:
```suggestion
        visible={showModalUpdate}
```

File: app/src/scenes/ConsoFollowUp/ConsoFeedDisplay.js [hunk 5]
```diff
@@ -30,6 +30,9 @@ const ConsoFeedDisplay = ({
 
   const drink = consolidatedCatalog.find((catalogDrink) => catalogDrink.drinkKey === drinkKey);
   const Icon = getIcon(drink.icon);
+  console.log('drink', drink);
+  console.log('Icon', Icon);
+  console.log('drinkName', drinkName);
```
Comment (arnaudambro) [inline app/src/scenes/ConsoFollowUp/ConsoFeedDisplay.js:35]:
```suggestion
```

**Pull request merged**